api/app/wildfire_one.py

```diff
@@ -132,25 +146,37 @@ def _is_station_valid(station) -> bool:
     return True
 
 
+def fetch_ecodivision_name(latitude: str, longitude: str, ecodivisions: geopandas.GeoDataFrame):
+    """ Returns the ecodivision name for a given lat/long coordinate """
+    # if station's latitude >= 60 (approx.), it's in the Yukon, so it won't be captured
+    # in the shapefile, but it's considered to be part of the SUB-ARCTIC HIGHLANDS ecodivision.
```

Comment:
I trust you, but where does this definition come from?

Reply:
it's come up before when we were loading stations from a static json file - somewhere (I don't remember where we've stored it) there's a map of how BC is split up into ecodivisions - as per our previous PO, the Sub-Arctic ecodivision (the northernmost in BC) should just be assumed to extend into the Yukon